app/assets/javascripts/discourse/controllers/create-account.js.es6

```diff
@@ -24,7 +30,7 @@ export default Ember.Controller.extend(
     accountPasswordConfirm: 0,
     accountChallenge: 0,
     formSubmitted: false,
-    rejectedEmails: Ember.A([]),
+    rejectedEmails: A([]),
```

Comment:
Slightly torn on this. I kinda prefer `Ember.A()` to the one-letter `A()` method.

Reply:
I wont fight for it :p will remove 👍